spec/models/conversion_host_spec.rb

```diff
@@ -4,8 +4,8 @@
   let(:apst) { FactoryBot.create(:service_template_ansible_playbook) }
 
   context "provider independent methods" do
-    let(:host) { FactoryBot.create(:host) }
-    let(:vm) { FactoryBot.create(:vm_or_template) }
+    let(:host) { FactoryBot.create(:host_redhat) }
```

Comment:
This set of specs is independent of providers and tying these host/vm to redhat/openstack defeats the purpose. can you keep the generic types and just mock them to return true for support check?

Reply:
Hm, this is somewhat problematic, because the `FactoryBot.create` method itself becomes an issue. I'll have to disable validations on the factories.

Reply:
Ok, restored, but I had to disable the conversion host factory validation.